behat-tests/README.txt

```diff
@@ -9,43 +9,44 @@ To set up your environment to run tests locally:
 1.  Download the Selenium stand-alone server JAR file:
     http://selenium-release.storage.googleapis.com/2.44/selenium-server-standalone-2.44.0.jar
 
-2.  Install Composer:
+2.  Selenium version 2.44 is only compatible with Firefox 33, but not 34 or above. You can download
```

Comment:
Of course - downloading selenium versions and playing with that is also beyond the scope of this ticket.  Perhaps we could just alert Bryan.